Python/Product/Analysis/Projects/PythonProject.cs

```diff
@@ -58,9 +55,14 @@ public abstract class PythonProject {
         /// <summary>
         /// Gets the current analyzer for the project, or null if no analyzer is available.
         /// </summary>
-        public abstract ProjectAnalyzer Analyzer {
-            get;
-        }
+        [Obsolete("Use the async version if possible")]
+        public abstract ProjectAnalyzer Analyzer { get; }
```

Comment:
Is this a public API?

Reply:
Yes, and formally public as well (unlike some of the "accidentally" public ones). The whole point of the abstract base class is to make it available.